.vscode/settings.json

```diff
@@ -61,11 +60,15 @@
     },
     "pwsh": null
   },
+  
+  "terminal.integrated.profiles.windows": {
+      "Git Bash": null
+  }
+    
 
-  // "terminal.integrated.defaultProfile.linux": "bash",
-
+  // The default terminal profile on Linux.
+  "terminal.integrated.defaultProfile.linux": "fish",
```

Comment:
```suggestion
  "terminal.integrated.defaultProfile.linux": "fish",
  "terminal.integrated.defaultProfile.windows": "PowerShell",
```

Reply:
id rarely expect to actually use pwsh tho:

- https://code.visualstudio.com/docs/remote/ssh